README.md

```diff
@@ -16,14 +58,66 @@ docker run \
 -v /var/run/docker.sock:/var/run/docker.sock \
 -v D:\host\path\where\to\output\singularity\image:/output \
 --privileged -t --rm \
-singularityware/docker2singularity \
+singularityware/docker2singularity:2.4 \
```

Comment:
I used tags so far to denote the version of docker client inside the container (see below - the client version needed to be compatible with the host version). I don't think we need to add tags here for singularity version. After this is merged I will make sure all branches will get updated.

Reply:
ok I'll remove that